src/module/meme-object.js

```diff
@@ -5,7 +5,13 @@ const setLocal = () => localStorage.setItem('memelist', JSON.stringify(memeList)
 // === get local storage === //
 const getLocal = () => JSON.parse(localStorage.getItem('memelist'));
```

Comment:
- [ ] This function is trying to get data from the local storage but there is an issue, If a user runs your project and this function will be called but on the user's browser there's no data and it will return null so index.js line 12 will throw an error, no single functionality will work because that forEach loop can not iterate a null value. You must check if there's data on the browser before returning that data. Please fix this issue.
 - To fix this you can consider to replace the function with this code.
 `const getLocal = () => {
  const data = JSON.parse(localStorage.getItem('memelist'));
  if(data !=null){
    return data;
  } else {
    return []
  }
};`
  
### See the image of the error.

![image](https://user-images.githubusercontent.com/83803414/160410855-889313cb-458d-4c36-9720-9ad239459179.png)

Reply:
thank you for mention the solution 😄